lib/src/main/java/com/github/kevinsawicki/http/HttpRequest.java

```diff
@@ -3178,6 +3178,25 @@ public HttpRequest form(final Map<?, ?> values, final String charset)
   }
 
   /**
+   * Write the json string
+   *
+   * @param jsonString
+   * @return this request
+   * @throws HttpRequestException
+   */
+  public HttpRequest json(final String jsonString)
+      throws HttpRequestException {
+    contentType(CONTENT_TYPE_JSON);
+    try {
+      openOutput();
```

Comment:
Is this a PR worth adding? If so, I'll update the code. Otherwise, no offense taken. 

When I first submitted it, I didn't realize that it could be sent using a combination of 'contentType' and `send` and thought I would need to `openOutput` myself. `form` caught my attention and got fixated on that.